manim/_config/default.cfg

```diff
@@ -194,6 +194,7 @@ repr_number = green
 # Uncomment the following line to manually set the loglevel for ffmpeg. See
 # ffmpeg manpage for accepted values
 loglevel = ERROR
+path = ffmpeg
```

Comment:
What does this `path` key mean? Can you add some docs on them either here as comments or in the tutorial for the configuration page?

Does this mean the path to the executable of FFmpeg, in which case I would suggest renaming it to `executable`.

Reply:
Again, I would like this to be changed.

Reply:
I support this, `executable` is less ambiguous than `path`.